tilequeue/command.py

```diff
@@ -2468,6 +2468,27 @@ def command_fn(cfg, args):
                            help='Tile coordinate as "z/x/y".')
     subparser.add_argument('--run_id', required=False,
                            help='optional run_id used for logging')
+    subparser.add_argument('--postgresql_host', required=False,
```

Comment:
If this is a list, this param should be called `postgresql_hosts`?

Reply:
> If this is a list, this param should be called `postgresql_hosts`?

yup I will change to plural form, I just mimic the existing environment variable naming.